### DISCUSSION TITLE
window.location.hash changed after click the link in Getting Started page.

### DISCUSSION THREAD

**Discussion by waltyou** [eclipse-theia/theia#9316] [Q&A] [answered]
Just like the images showing, after i click the `Open` link in Getting Started Page, the `window.location.hash` changed to 'http://localhost:3000/#' from 'http://localhost:3000/#/home/theia' in Chrome.

![image](https://user-images.githubusercontent.com/8199399/113974014-cfcc5a00-986f-11eb-8fb7-8c2b8fddc4c5.png)

![image](https://user-images.githubusercontent.com/8199399/113973825-7fed9300-986f-11eb-92f4-76fde5af2dd0.png)

Do you know why this happened? and how to avoid `window.location.hash` change?

The Theia version is v1.12.1.

**Comment by vince-fugnitto** [comment, marked as answer]
@waltyou thank you for the discussion. We should update the implementation of the `getting-started` view to remove uses of `href="#"` which will modify the URL hash, or convert them to buttons.

For example:

https://github.com/eclipse-theia/theia/blob/6f7f401ae299e2cf25c06566f8b5b0a0d41f33f8/packages/getting-started/src/browser/getting-started-widget.tsx#L159-L162

**Reply by vince-fugnitto** [reply]
#9318

**Reply by waltyou** [reply]
@vince-fugnitto Got it. Very appreciate your help!
